packages/client-features/src/utils.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Helper function to check if we're in the block editor
+ */
+export const isInEditor = (): boolean => {
+	try {
+		return !! document.querySelector( '.block-editor' );
```

Comment:
How is the Editor or Block Editor context defined? Is it limited to the post editor, which contains the class `.block-editor`, or is the Site Editor, which doesn't contain that class, in scope too?

Reply:
Good call out.

I've updated the PR to have separate methods now: `isInPostEditor`, `isInSiteEditor`, and `isInEditor`. Most features can be exposed for both editors, with the exception of the post title one.